astro/src/content/quickstarts/quickstart-dotnet-api.mdx

```diff
@@ -0,0 +1,445 @@
+
+<Aside type="note">
+If you simply want to run the application, there is a completed version in the `complete-application` directory. You can use the following commands to get it up and running if you do not want to create your own.
+
+```shell
+cd complete-application
+dotnet run
+```
+
+You can then follow the instructions in the [Run the API](./#run-the-api) section.
+</Aside>
+
+Now, create a base ASP.NET API project. 
+
+```shell
+dotnet new webapi -o your-api && cd your-api
+```
+
+### Add Security
+
+This app uses the standard [.NET JWTBearer package](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.JWTBearer), which simplifies extracting and validating JWT tokens from FusionAuth.
+
+Install the .NET JWTBearer package and other supporting libraries. 
+
+```shell
+dotnet add package Microsoft.AspNetCore.Authentication.JwtBearer
+dotnet add package Microsoft.AspNetCore.OpenApi
+dotnet add package System.IdentityModel.Tokens.Jwt
+dotnet add package Microsoft.IdentityModel.Protocols.OpenIdConnect
+```
+
+Next, update your `appsettings.json` file with settings specific to the FusionAuth Issuer and Client Id (Audience).
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-dotnet-api/main/complete-application/appsettings.json"/>
+
+Replace the code in the `appsettings.Development.json` file with the following.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-dotnet-api/main/complete-application/appsettings.Development.json"/>
+
+The `appsettings.Development.json` file will be used when you run the application in development mode. It will enable some useful development options, mainly that HTTPS will not be required for the Issuer (FusionAuth) server. In production, you must use HTTPS for the Issuer server.
+
+Update the `Program.cs` file to configure ASP.NET authentication to use JWT tokens. Replace the code in the `Program.cs` file with the following code.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-dotnet-api/main/complete-application/Program.cs"
+lang="c#" />
+
+The `Program.cs` file configures the authentication services to use JWT bearer tokens to authenticate users through the `.AddJwtBearer()` method. The Issuer and Audience settings used by the JWT verification code are set in the `appsettings.json` file under the `Authentication` section. Note the `options.Events` section that configures the JWT bearer authentication to use the `OnMessageReceived` event to read the JWT from the `app.at` cookie if it is present. By default, the JWT bearer authentication will only read the JWT from the `Authorization` header. With this code, the JWT can be passed in either the `Authorization` header or the `app.at` cookie.
```

Comment:
The "Note the.." should either be in an Aside or just remove "Note" I.E. The `options.Events` section...

Reply:
removed "Note"